cypress/integration/login.js

```diff
@@ -20,5 +20,7 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        var d = new Date()
+        cy.get('.calendar--today > span').should('have.text', d.getDate().toString())
```

Comment:
".calendar--today" is enough to find this locator, no need to chain it to span, as it create additional dependency